config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,18 @@
+PUPPET_CLASS = 'foreman_proxy::plugin::pulp'
+PARAM = 'pulpcore_enabled'
+
+def upgrade
+  answers[PUPPET_CLASS][PARAM] = true
+  return unless answers['katello']
+  answers['katello'] = {} unless answers['katello'].is_a?(Hash)
+  answers['katello']['use_pulp_2_for_file'] = true
+  answers['katello']['use_pulp_2_for_docker'] = true
+  answers['foreman_proxy_content']['proxy_pulp_isos_to_pulpcore'] = false
```

Comment:
This needs a similar guard as lines 6 and 7 for `katello` to ensure it's safe. It's unlikely, but I like to be defensive in my code.

Reply:
I've created separate `unless` blocks for `katello` and `foreman_proxy_content`. Thanks